react/store/createReturnRequest/components/ItemsDetails.tsx

```diff
@@ -51,6 +51,7 @@ export const ItemsDetails = (itemToReturn: ItemToReturn) => {
         ...item,
         returnReason: {
           reason,
+          otherReason,
```

Comment:
I would only add this property if the reason is otherReason. This way, we keep the pattern as: If the reason is different than `otherReason`, there is no `otherReason` property on the object. The way we designed here, some can have it as an empty string, other will not have it.

```suggestion
          ...(reason === 'otherReason' ? { otherReason } : null),
```